helm-charts/splunk-otel-collector/templates/config/_otel-agent.tpl

```diff
@@ -381,6 +381,9 @@ receivers:
     # use the read-only endpoint instead.
     auth_type: none
     endpoint: ${K8S_NODE_IP}:10255
+    {{ else if eq .Values.distribution "aks" }}
+    ca_file: "/hostfs/etc/kubernetes/certs/kubeletserver.crt"
```

Comment:
Do we have to wait for 0.124 collector version for this config to be available in the kubeletstats receiver?

Reply:
`0.124` upstream was released yesterday, are we planning a chart release before we update our splunk distro to `0.124` ?

Reply:
The chart latest is 0.122 collector. I think we'll make a chart release for collector 0.123. I would recommend marking this as draft and once the appVersion is bumped to 0.124, we merge this before releasing the chart for the same (the failing aks test will also pass then).